src/translations/en.json

```diff
@@ -27,7 +27,8 @@
   "error": {
     "pageNotFound": "Page not found",
     "appNotFound": "App not found",
-    "transactionFailedReasonUnknown": "Transaction failed: reason unknown"
+    "transactionFailedReasonUnknown": "Transaction failed: reason unknown",
```

Comment:
Just a note for the future - we'll change this to give the user an instruction on how to try to continue